bot/internal/env/env.go

```diff
@@ -132,6 +132,14 @@ func (e *Environment) IsCloudDeployBranch() bool {
 		(e.UnsafeBase == CloudProdBranch || e.UnsafeBase == CloudStagingBranch)
 }
 
+// Team returns CloudTeam when the repository is the cloud repo otherwise CoreTeam.
+func (e *Environment) Team() string {
```

Comment:
```suggestion
func (e *Environment) RepoOwner() string {
```

Perhaps?

Reply:
Changed to `RepoOwnerTeam` since it returns a `Team` name.